### PR TITLE
fix(codegen): from json file

### PR DESCRIPTION
**Pull request by froid1911**

### CODE REVIEW

File: packages/codegen/src/codegen/utils.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export async function loadDocumentModel(
       documentModel = file.state.global;
     } else if (path.endsWith(".json")) {
       const data = fs.readFileSync(path, "utf-8");
-      const document = JSON.parse(data) as DocumentModelState;
```
Comment (acaldas) [inline packages/codegen/src/codegen/utils.ts:16]: This doesn't break the zip loading?

Comment (froid1911) [reply packages/codegen/src/codegen/utils.ts:16]: only if the zip file ends with .json, see line 14

**Comment by acaldas** [comment]
Nice, there just seems to an issue with the commas.

Screenshot 2024-10-30 at 15 57 31

File: packages/codegen/src/codegen/utils.ts (hunk 1 above)
Comment (acaldas) [reply packages/codegen/src/codegen/utils.ts:16]: Nice, there is an eslint error due to data being any.

Comment (acaldas) [reply packages/codegen/src/codegen/utils.ts:16]: Ah I think this was done to support generating from the spec.json file. I think we need a check that gets the document state on both cases.

**Pull request merged**